### PR TITLE
DRAFT: [BD-26] [EDUCATOR-5813] Add prerequisites check in the MFE learning app API before user starts a proctored exam

### PR DESCRIPTION
**Pull request by ViktorRusakov**
**Description:**

Extend MFE learning app API to check if prerequisites are satisfied before user starts a proctored exam.

NOTE: should be merged after #864 

**JIRA:**

[EDUCATOR-5813](https://openedx.atlassian.net/browse/EDUCATOR-5813)

**Pre-Merge Checklist:**

- [ ] Updated the version number in `edx_proctoring/__init__.py` and `package.json` if these changes are to be released.
- [x] Described your changes in `CHANGELOG.rst`
- [ ] Confirmed Github reports all automated tests/checks are passing.
- [ ] Approved by at least one additional reviewer.

**Post-Merge:**

- [ ] Create a tag matching the new version number.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @ViktorRusakov! I've created [BLENDED-863](https://openedx.atlassian.net/browse/BLENDED-863) to keep track of it in Jira.   More details are on the [BD-26](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2523627980/BD-26+Proctoring+MFE+and+configuration) project page.


When this pull request is ready, tag your edX technical lead.

File: edx_proctoring/tests/test_mfe_views.py [hunk 1]
```diff
@@ -63,6 +71,9 @@ def test_get_started_proctored_exam_attempts_data(self):
         self.assertEqual(response.status_code, 200)
         response_data = json.loads(response.content.decode('utf-8'))
         exam_data = response_data['exam']
+
+        # if exam started we do not check for prerequisites
```
Comment (idegtiarov) [inline edx_proctoring/tests/test_mfe_views.py:75]: Nit 
```suggestion
        # if exam started the prerequisites are not checked
 ```

File: edx_proctoring/views.py [hunk 2]
```diff
@@ -227,6 +230,9 @@ def get(self, request, course_id, content_id):
                         attempt.get('id'),
                         is_learning_mfe=is_learning_mfe
                     )
+                # exam hasn't started yet and is proctored so check if prerequisites are satisfied
```
Comment (idegtiarov) [inline edx_proctoring/views.py:233]: Nit: 
```suggestion
                # exam hasn't been started yet but it is proctored so needed to be checked if prerequisites are satisfied
```

**Review by idegtiarov — CHANGES_REQUESTED** [review]
Minor nits in the comments

**Pull request closed without merging**